Observatory-Control-System/EAS/DIMM/ops.rst

```diff
@@ -194,5 +210,25 @@ and within the OpenTPL:
 
 Confirm in the monitors that the DIMM operations have ceased. 
 
+Sometimes it may be necessary to manually issue commands to the DIMM dome to close. This can be done by issuing commands to *tt-dome*
+
+.. code-block:: bash
+   
+        dimm@dimm:~$  telnet 127.0.0.1 16302
+
+.. code-block:: bash
+   
+        AUTH OK 0 0
+        1 run sidea=0.0
+        2 run sideb=0.0
+
+If there is an issue, it is common to need to reconnect to *tt-dome* through telnet and issue the commands again.
+
+You can check the dome position with the following commands to *tt-dome*
```

Comment:
Here also declare what the result of position means: 0 closed, near 1 open as the sidea and sideb never reach 1, and stay around 0.8.